core/src/main/java/io/servicecomb/core/CseApplicationListener.java

```diff
@@ -124,9 +129,26 @@ public void onApplicationEvent(ApplicationEvent event) {
           schemaListenerManager.notifySchemaListener();
 
           triggerEvent(EventType.BEFORE_REGISTRY);
+          /*
```

Comment:
it's better to write new comments in english.
and maybe this logic can be a new method, only invoke in this place.

Reply:
I've extracted this part into a new private method, and translated the comments into English. Please review it.